app/serializers/rest/admin/moderation_note_serializer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class REST::Admin::ModerationNoteSerializer < ActiveModel::Serializer
```

Comment:
The idea behind this is to be able to reuse it for ReportNotes, AccountModerationNotes and InstanceNotes, as these are all effectively the same, the only thing that varies is the object that they target.